src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs

```diff
@@ -75,34 +75,101 @@ public async Task Get()
                 }
                 catch (Exception ex)
                 {
-                    if (Logger.IsInfoEnabled)
-                        Logger.Info("Error encountered in cluster dashboard handler", ex);
+                    await HandleException(ex, webSocket);
+                }
+            }
+        }
 
-                    try
+        [RavenAction("/cluster-dashboard/remote/watch", "GET", AuthorizationStatus.ClusterAdmin, SkipUsagesCount = true)]
```

Comment:
Need `admin` in the link, no?

Reply:
What happens if we aren't secured?

Reply:
IIRC everyone is cluster admin in unsecured?

Reply:
1. Added `admin` in https://github.com/ravendb/ravendb/pull/12106/commits/36c23bea0dd0abcccd10625f9588b51b8260a40d
2. Correct. Then we're ClusterAdmin by default